corehq/apps/locations/tests/test_permissions.py

```diff
@@ -234,8 +243,31 @@ def test_can_edit_worker(self):
     def test_cant_edit_worker(self):
         self._assert_edit_user_gives_status(self.cambridge_worker, 404)
 
+    def _call_djangoRMI(self, url, method_name, data):
+        data = json.dumps(data)
+        return self.client.post(
+            url,
+            content_type="application/json;charset=utf-8",
+            **{
+                'HTTP_DJNG_REMOTE_METHOD': method_name,
+                'HTTP_X_REQUESTED_WITH': 'XMLHttpRequest',
+                'CONTENT_LENGTH': len(data),
+                'wsgi.input': StringIO(data),
+            })
```

Comment:
This magic combination of parameters and serialization of data was incredibly obnoxious to figure out.  If anyone else ever needs to test djangoRMI methods, hopefully they'll find this.